src/app/app.module.ts

```diff
@@ -16,10 +18,12 @@ export function HttpLoaderFactory(http: HttpClient) {
 
 @NgModule({
   declarations: [
-    AppComponent
+    AppComponent,
+    FooterComponent
```

Comment:
As the FooterComponent is part of the layout, you should add it to the LayouModule instead of the AppModule